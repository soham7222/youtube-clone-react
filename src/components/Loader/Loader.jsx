import React from 'react'
import "./Loader.css"

export const Loader = () => {
    return (
        <div id='preloader'>
            <div className='spinner'>
                <span className='ball-1'></span>
                <span className='ball-2'></span>
                <span className='ball-3'></span>
                <span className='ball-4'></span>
                <span className='ball-5'></span>
                <span className='ball-6'></span>
                <span className='ball-7'></span>
                <span className='ball-8'></span>
            </div>
        </div>
    )
}
