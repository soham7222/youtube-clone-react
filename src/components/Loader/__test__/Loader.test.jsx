import React from 'react';
import { render } from '@testing-library/react';
import { Loader } from '../Loader';

it('Loader renders correctly', () => {
    const dom = render(<Loader />)
    expect(dom).toMatchSnapshot();
});