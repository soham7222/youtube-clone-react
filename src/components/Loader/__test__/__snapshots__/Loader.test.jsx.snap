// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        id="preloader"
      >
        <div
          class="spinner"
        >
          <span
            class="ball-1"
          />
          <span
            class="ball-2"
          />
          <span
            class="ball-3"
          />
          <span
            class="ball-4"
          />
          <span
            class="ball-5"
          />
          <span
            class="ball-6"
          />
          <span
            class="ball-7"
          />
          <span
            class="ball-8"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      id="preloader"
    >
      <div
        class="spinner"
      >
        <span
          class="ball-1"
        />
        <span
          class="ball-2"
        />
        <span
          class="ball-3"
        />
        <span
          class="ball-4"
        />
        <span
          class="ball-5"
        />
        <span
          class="ball-6"
        />
        <span
          class="ball-7"
        />
        <span
          class="ball-8"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
